docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -20,8 +20,9 @@ If a virtual machine or a physical server in a universe reaches its end of life
 When this happens, new Master leaders are elected for the underlying data shards, but since the universe enters a partially under-replicated state, it would not be able to tolerate additional failures. To remedy the situation, you can eliminate the unreachable node by taking actions in the following sequence:
 
 - Step 1: [Remove node](#remove-node)
-- Step 2: [Release instance](#release-instance)
-- Step 3: [Delete node](#delete-node)
+- Step 2: [Start a new master](#start-a-new-master) if necessary
```

Comment:
Shouldn't this content be in [Start a process](https://docs.yugabyte.com/preview/yugabyte-platform/manage-deployments/start-stop-processes/#start-a-process) ?
We can include a link to it as a step in Remove nodes.

Reply:
What we want here is a one-stop runbook for users to remove a "dead node", even if it means duplicating some information.